# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# goal definition
geometry_msgs/PoseStamped dogPose
geometry_msgs/PointStamped goalPosition
